libs/ledger-live-common/src/families/cardano/speculos-deviceActions.ts

```diff
@@ -1,16 +1,17 @@
 import type { DeviceAction } from "../../bot/types";
 import type { Transaction } from "./types";
 import { deviceActionFlow, formatDeviceAmount } from "../../bot/specs";
+import { getAccountStakeCredential, getBipPathString } from "./logic";
 
 export const acceptTransaction: DeviceAction<Transaction, any> =
   deviceActionFlow({
     steps: [
       {
-        title: "Start new",
+        title: "New ordinary",
       },
       {
-        title: "ordinary transaction?",
-        button: "Rr",
+        title: "transaction?",
+        button: "LRlr",
```

Comment:
The `button` field has recently been typed on our side, just make sure to use the according types when you'll rebase.